bib/cmd/bootc-image-builder/main.go

```diff
@@ -422,6 +424,7 @@ func run() error {
 	manifestCmd.Flags().String("rpmmd", "/rpmmd", "rpm metadata cache directory")
 	manifestCmd.Flags().String("target-arch", "", "build for the given target architecture (experimental)")
 	manifestCmd.Flags().StringArray("type", []string{"qcow2"}, "image types to build [qcow2, ami, iso, raw]")
+	manifestCmd.Flags().Bool("local", false, "use a local container rather than a container from a registry")
```

Comment:
Do we need a local flag, or should BIB always look for a local image and then pull the image if it is not in local storage?

Reply:
Is that your preference? I think I would like that as well, but we wanted to do the minimal change for now, and refactor it later.

Reply:
I just don't see why I would volume mount the images in and then not use local. Also I think we should allow non local pulls when storage is mounted.  One image might require many layers that are already stored in the local storage but have different content in the upper layers.

Reply:
Defaulting to local storage will be a change for consumers, but I think it's the right time to make that change, and we add a `--remote` flag for the previous default.

Reply:
Sounds sane. Just to double-check: Is it fine that bib will use different versions of podman/skopeo to manipulate the store than the host has? In other words, the format of the containers store is stable, right?

Reply:
Yes it has been stable. Definitely forward stable.  But I doubt backwards would be an issue.  Besides we are just talking about images here not containers. When we to to using composefs we might need to make sure we don't have issues.

@giuseppe Thoughts?